src/clusterfuzz/_internal/fuzzing/fuzzer_selection.py

```diff
@@ -118,11 +118,11 @@ def get_fuzz_task_payload(platform=None):
   if base_platform != platform:
     platforms.append(base_platform)
 
-  query = data_types.FuzzerJobs.query()
```

Comment:
can you explain more the context behind this change?

`FuzzerJobs` may not exist locally because these are created by this cron job: https://github.com/google/clusterfuzz/blob/b3b383c701a2b3a0cae4eaf99657311ac3d0d27b/src/clusterfuzz/_internal/cron/batch_fuzzer_jobs.py#L24 

This cron job just batches `FuzzerJob` entities into `FuzzerJobs` for efficiency reasons.

Reply:
Yes, please see [this change](https://github.com/google/clusterfuzz/commit/f94ad31a2f9f7ae3175c4c1f14d6374b1e0ea85e#diff-e3b3935eaad6337aed2c5661b9015b02405fce8ba8a86f1137f187131fcc1036R121) for more context.

Basically I am reverting it to how it used to be before the above change, because now the query returns no data (only when developing locally) due to the `FuzzerJob` -> `FuzzerJobs` change.

Reply:
Got it, thanks!